src/BackpackServiceProvider.php

```diff
@@ -4,6 +4,7 @@
 
 use Backpack\CRUD\app\Http\Middleware\ThrottlePasswordRecovery;
 use Backpack\CRUD\app\Library\CrudPanel\CrudPanel;
+use Backpack\CRUD\app\Library\CrudPanel\ViewNamespaces;
```

Comment:
How about 

```suggestion
use Backpack\CRUD\ViewNamespaces;
```

after all, namespaces don't really only concern CrudPanel elements, they concern widgets too, right? When we'll separate this package into Core and CRUD... we'll move it in Core, right?